TACC/Core-Portal
# Task/FP-827 - Assorted Dashboard Fixes

## Overview: ##

- Add Ticket Modal now has a footer (still has a scrolling issue that will be addressed in [FP-344](https://jira.tacc.utexas.edu/browse/FP-344)
- Added space between submit and spinner elements on the feedback modal
- Edit Optional/Required Information modals have submit button right aligned.
- Fixed spacing issue with btn-link in "Request Activation" modal for MATLAB 3rd party app "here" link

## Related Jira tickets: ##

* [FP-827](https://jira.tacc.utexas.edu/browse/FP-827)


## Testing Steps: ##
Navigate to the dashboard and check the following modals:
- Add Ticket
- Leave Feedback
- Edit Optional/Required Information
- Request Activation

## UI Photos:
<img width="716" alt="ui_changes1" src="https://user-images.githubusercontent.com/29575979/106031591-142fc280-6095-11eb-8227-6a445ce4f955.png">
<img width="653" alt="ui_changes2" src="https://user-images.githubusercontent.com/29575979/106031600-15f98600-6095-11eb-9b68-e8acd3bef168.png">
<img width="671" alt="ui_changes3" src="https://user-images.githubusercontent.com/29575979/106031602-15f98600-6095-11eb-9935-1d4cef258961.png">
<img width="672" alt="ui_changes4" src="https://user-images.githubusercontent.com/29575979/106031604-16921c80-6095-11eb-9d3a-f944bf508976.png">
<img width="658" alt="ui_changes5" src="https://user-images.githubusercontent.com/29575979/106031607-16921c80-6095-11eb-87d6-d101f7dbfc3f.png">
<img width="668" alt="ui_changes6" src="https://user-images.githubusercontent.com/29575979/106031610-172ab300-6095-11eb-9d2a-008d23111d26.png">
<img width="704" alt="ui_changes7" src="https://user-images.githubusercontent.com/29575979/106031612-172ab300-6095-11eb-8b85-05e26eea8e74.png">
